apps/ocpp/call_handlers/errors/__init__.py

```
@@ -0,0 +1,164 @@
+    "handle_get_charging_profiles_error",
+    "handle_get_display_messages_error",
+    "handle_get_report_error",
+    "handle_set_display_message_error",
+    "handle_set_monitoring_base_error",
+    "handle_set_monitoring_level_error",
+    "handle_set_network_profile_error",
+    "handle_install_certificate_error",
+    "handle_delete_certificate_error",
+    "handle_certificate_signed_error",
+    "handle_get_installed_certificate_ids_error",
+]
```

Ruff (0.14.14)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)